torchgeo/datasets/flair2.py

```diff
@@ -0,0 +1,762 @@
+        },
+    }
+    globs: ClassVar[dict[str, str]] = {
+        'images': 'IMG_*.tif',
+        'sentinels': 'SEN2_*{0}.npy',
+        'masks': 'MSK_*.tif',
+    }
+    centroids_file: str = 'flair-2_centroids_sp_to_patch'
+    # Sentinel super patch size according to datapaper
+    super_patch_size: int = 40
+
+    # Band information
+    rgb_bands: tuple[str, str, str] = ('B01', 'B02', 'B03')
+    all_bands: tuple[str, str, str, str, str] = ('B01', 'B02', 'B03', 'B04', 'B05')
+
+    # Note: the original dataset contains 18 classes, but the dataset paper suggests
+    # grouping all classes >13 into "other" class, due to underrepresentation
+    classes: tuple[str, ...] = (
+        'building',
+        'pervious surface',
+        'impervious surface',
+        'bare soil',
+        'water',
+        'coniferous',
+        'deciduous',
+        'brushwood',
+        'vineyard',
+        'herbaceous vegetation',
+        'agricultural land',
+        'plowed land',
+        'other',
+    )
+    # Define a colormap for the classes
+    cmap = ListedColormap(
+        [
+            'cyan',  # building
+            'lightgray',  # pervious surface
+            'darkgray',  # impervious surface
+            'saddlebrown',  # bare soil
+            'blue',  # water
+            'darkgreen',  # coniferous
+            'forestgreen',  # deciduous
+            'olive',  # brushwood
+            'purple',  # vineyard
+            'lime',  # herbaceous vegetation
+            'yellow',  # agricultural land
+            'orange',  # plowed land
+            'red',  # other
+        ]
+    )
+
+    statistics: ClassVar[dict[str, dict[str, dict[str, float]]]] = {
+        'train': {
+            'B01': {
+                'min': 0.0,
+                'max': 255.0,
+                'mean': 113.77526983072,
+                'stdv': 1.4678962001526,
+            },
+            'B02': {
+                'min': 0.0,
+                'max': 255.0,
+                'mean': 118.08112962721,
+                'stdv': 1.2889349378677,
+            },
+            'B03': {
+                'min': 0.0,
+                'max': 255.0,
+                'mean': 109.27393364381,
+                'stdv': 1.2674219560871,
+            },
+            'B04': {
+                'min': 0.0,
+                'max': 255.0,
+                'mean': 102.36417944851,
+                'stdv': 1.1057592647291,
+            },
+            'B05': {
+                'min': 0.0,
+                'max': 255.0,
+                'mean': 16.697295721745,
+                'stdv': 0.82764953440507,
+            },
+        }
+    }
+
+    @staticmethod
+    def per_band_statistics(
+        split: str, bands: Sequence[str] = all_bands
+    ) -> tuple[list[float], ...]:
+        """Get statistics (min, max, means, stdvs) for each used band in order.
+
+        Args:
+            split (str): Split for which to get statistics (currently only for train)
```

Comment:
For the docstring notion in torchgeo, we do not include the type in the docstring again, only function arguments.

Reply:
I think this will also resolve the failing docs test